app/cells/date_range_cell.rb

```diff
@@ -20,16 +20,22 @@ def date_field(type)
 
   def select_div
     options[:form].send(:form_group_builder, :daterange, label: label) do
-      content_tag(
-        :div,
-        '',
-        class: 'ui-components-date-range form-control',
-        data: { start: "##{id}_from", end: "##{id}_to" }
-      )
+      content_tag(:div, '', class: 'ui-components-date-range form-control', data: data)
     end
   end
 
+  def data
+    options.slice(:ranges, :date_limit)
+      .transform_values { |v| serialize(v) }
+      .merge(start: "##{id}_from", end: "##{id}_to")
+      .merge(options.slice(:start_date, :end_date).transform_values(&:to_s))
+  end
+
   def id
     "date_range_#{name_param}"
   end
+
+  def serialize(value)
+    ERB::Util.html_escape(value.to_json)
```

Comment:
`cells-slim` was a different version on CircleCI and on my machine. I just did `bundle update cells-slim` and it updated from 0.0.2 to 0.0.3, which it already is on CirlceCI. How is this even possible?